src/fsipy/simulations/cylinder.py

```diff
@@ -1,3 +1,223 @@
+    # Overwrite default values
+    E_s_val = 1e6  # Young modulus (Pa)
+    nu_s_val = 0.45
+    mu_s_val = E_s_val / (2 * (1 + nu_s_val))  # 0.345E6
+    lambda_s_val = nu_s_val * 2.0 * mu_s_val / (1.0 - 2.0 * nu_s_val)
+
+    default_variables.update(
+        dict(
+            # Temporal parameters
+            T=0.1,  # Simulation end time
+            dt=0.001,  # Time step size
+            theta=0.501,  # Theta scheme (implicit/explicit time stepping)
+            save_step=1,  # Save frequency of files for visualisation
+            checkpoint_step=50,  # Save frequency of checkpoint files
+            # Linear solver parameters
+            linear_solver="mumps",
+            atol=1e-6,  # Absolute tolerance in the Newton solver
+            rtol=1e-6,  # Relative tolerance in the Newton solver
+            recompute=20,  # Recompute the Jacobian matix within time steps
+            recompute_tstep=20,  # Recompute the Jacobian matix over time steps
+            # boundary condition parameters
+            mesh_path="artery_coarse_rescaled",
+            inlet_id=2,  # inlet id for the fluid
+            inlet_outlet_s_id=11,  # inlet and outlet id for solid
+            fsi_id=22,  # id for fsi interface
+            rigid_id=11,  # "rigid wall" id for the fluid and mesh problem
+            outer_wall_id=33,  # id for the outer surface of the solid
+            # Fluid parameters
+            rho_f=1.025e3,  # Fluid density [kg/m3]
+            mu_f=3.5e-3,  # Fluid dynamic viscosity [Pa.s]
+            dx_f_id=1,  # ID of marker in the fluid domain. When reading the mesh, the fuid domain is assigned with a 1.
+            v_max_final=0.75,  # Steady state, maximum centerline velocity of parabolic profile
+            P_final=10000,  # Steady State pressure applied to wall
+            # Solid parameters
+            rho_s=1.0e3,  # Solid density [kg/m3]
+            mu_s=mu_s_val,  # Solid shear modulus or 2nd Lame Coef. [Pa]
+            nu_s=nu_s_val,  # Solid Poisson ratio [-]
+            lambda_s=lambda_s_val,  # Solid 1st Lame Coef. [Pa]
+            dx_s_id=2,  # ID of marker in the solid domain
+            # mesh lifting parameters (see turtleFSI for options)
+            extrapolation="laplace",  # laplace, elastic, biharmonic, no-extrapolation
+            extrapolation_sub_type="constant",  # ["constant","small_constant","volume","volume_change","bc1","bc2"]
+            folder="cylinder_test",  # output folder generated for simulation
+            kill_time=100000,  # in seconds, after this time start dumping checkpoints every timestep
+            save_deg=1  # save_deg=1 saves corner nodes only, save_deg=2 saves corner + mid-point nodes for viz
+        )
+    )
+
+    return default_variables
+
+
+def get_mesh_domain_and_boundaries(mesh_path, folder, **namespace):
+    print("Obtaining mesh, domains and boundaries...")
+    mesh = Mesh()
+    hdf = HDF5File(mesh.mpi_comm(), "mesh/" + mesh_path + ".h5", "r")
```

Comment:
```suggestion
    hdf = HDF5File(mesh.mpi_comm(), mesh_path, "r")
```
Instead of assuming that there’s a folder `mesh`, it would be more user-friendly to directly use path to the mesh.